### PR TITLE
Document ASDCONF and use it for initial check of config file existence

### PR DESCRIPTION
**Pull request by imsodin**
I wanted to start asd for different directories together with their corresponding services and was thus looking for an option to specify different config files. I found it was possible but only after looking at the code. I think this should be mentioned in the man page. I hope I added the comment about it at a sensible place.

### CODE REVIEW

File: common/anything-sync-daemon.in [hunk 1]
```diff
@@ -14,9 +14,16 @@ VERS="@VERSION@"
 
 ASDCONF=${ASDCONF:-"/etc/asd.conf"}
 DAEMON_FILE=${DAEMON_FILE:-"/run/asd"}
+if [[ "$ASDCONF" != "/etc/asd.conf" ]]; then
+    ASDNAME="${ASDCONF##*/}"
+    ASDNAME="-${ASDNAME%.conf}"
```
Comment (manorit2001) [inline common/anything-sync-daemon.in:19]:
```suggestion
    ASDNAME="${ASDNAME%.conf}"
```
I would prefer that the `ASDNAME` variable doesn't have prefix `-` in it. And the variables using it add that prefix in them. Like `TMP="$VOLATILE/asd-$ASDNAME-$USER$DIR"`

**Review by manorit2001 — COMMENTED** [review]
Hi, I know it's been too long but would you still be willing to work on this PR?

**Comment by manorit2001** [comment]
Awesome, could you please rebase on top of `dev` ?

File: common/anything-sync-daemon.in [hunk 2]
```diff
@@ -15,9 +15,18 @@ VERS="@VERSION@"
 ASDCONF=${ASDCONF:-"/etc/asd.conf"}
 DAEMON_FILE=${DAEMON_FILE:-"/run/asd"}
 CRASH_RECOVERY_SUFFIX="crashrecovery"
+ASDNAME="asd"
+if [[ "$ASDCONF" != "/etc/asd.conf" ]]; then
+    suffix="${ASDCONF##*/}"
+    suffix="-${ASDNAME%.conf}"
```
[the remark this answers is not in the document]
Comment (imsodin) [reply common/anything-sync-daemon.in:21]: I certainly wouldn't use only a hash, as it removes meaning from what's presented to the user, but e.g. an additional suffix then. However I'll defer to you for that, please use or discard the current state as you see fit.

**Pull request merged**